packages/console/src/pages/Applications/index.tsx

```diff
@@ -89,69 +85,50 @@ const Applications = () => {
           />
         </Modal>
       </div>
-      <StickyHeaderTable
+      <Table
         className={resourcesStyles.table}
-        header={
-          <thead>
-            <tr>
-              <th>{t('applications.application_name')}</th>
-              <th>{t('applications.app_id')}</th>
-            </tr>
-          </thead>
-        }
-        colGroup={
-          <colgroup>
-            <col className={styles.applicationName} />
-            <col />
-          </colgroup>
-        }
-      >
-        <tbody>
-          {!data && error && (
-            <TableError
-              columns={2}
-              content={error.body?.message ?? error.message}
-              onRetry={async () => mutate(undefined, true)}
-            />
-          )}
-          {isLoading && <TableLoading columns={2} />}
-          {applications?.length === 0 && (
-            <TableEmpty columns={2}>
-              <Button
-                title="applications.create"
-                type="outline"
-                onClick={() => {
-                  navigate({
-                    pathname: createApplicationPathname,
-                    search,
-                  });
-                }}
+        rowGroups={[{ key: 'applications', data: applications }]}
+        rowIndexKey="id"
+        isLoading={isLoading}
+        errorMessage={error?.body?.message ?? error?.message}
+        columns={[
+          {
+            title: t('applications.application_name'),
+            dataIndex: 'name',
+            colSpan: 6,
```

Comment:
Why is there a `colSpan: 6`?

Reply:
According to the design 
<img width="898" alt="image" src="https://user-images.githubusercontent.com/10806653/209960830-63f8d803-22ad-430a-bed6-448c0c2013b0.png">

Reply:
ok